python/agents/application/cron.py

```
@@ -0,0 +1,24 @@
+
+    def start(self) -> None:
+        while True:
+            self.schedule.exec_jobs()
+            time.sleep(1)
+
+
+class TradingAgent(Scheduler):
+    def __init__(self) -> None:
+        super()
+        self.trader = Trader()
+        self.weekly(Monday(), self.trader.one_best_trade)
```

Pylint (3.3.7)

[error] Instance of 'TradingAgent' has no 'weekly' member

(E1101)